variables.tf

```diff
@@ -26,6 +26,7 @@ variable "repos" {
       cascade_dependencies = optional(bool)
       filter               = optional(string)
       use_project_markers  = optional(bool)
+      checkov              = optional(bool, false)
```

Comment:
let's think for the future to not break api if we want to parametrise checkov per workflow in the future.
Add 
```
checkov = optional({
  enabled = optional(bool, false)
}, {})
```
instead.

Also this option should not be inside the `terragrunt_atlantis_config` block